airflow/auth/managers/fab/openapi/v1.yaml

```diff
@@ -0,0 +1,712 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+---
+openapi: 3.0.3
+
+info:
+  title: "Flask App Builder User & Role API"
+
+  version: '1.0.0'
+  license:
+    name: Apache 2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
+  contact:
+    name: Apache Software Foundation
+    url: https://airflow.apache.org
+    email: dev@airflow.apache.org
+
+paths:
+  /roles:
+    get:
+      summary: List roles
+      description: |
+        Get a list of roles.
+
+        *New in version 2.1.0*
```

Comment:
Should this be 2.8?

Reply:
Both are true I guess. This Rest API has been introduced in Airflow in 2.1.0 and now we are moving it to the FAB auth manager (which might happen in 2.8). But the API still exists in core Airflow and is redirected to this endpoint, thus saying `*New in version 2.8.0*` would be wrong. I'd be inclined to just remove the message "New in ..." to avoid any confusion. What do you think?

Reply:
But the old and new endpoints are both there right? Like yeah under the hood they are the same, but they are different endpoints from the clients perspective? Thus 2.8 here is the right answer?

Reply:
Oh you're right! I confused myself. Good point. Let me update that